### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by casperdcl**
possible fix to #32

### CODE REVIEW

**Comment by casperdcl** [comment]
issues: https://stackoverflow.com/questions/38085429/multiple-language-in-a-single-travis-yml-file

**Comment by casperdcl** [comment]
After fixing `USE_SSYTEM_SWIG` -> `USE_SYSTEM_SWIG`, we have an error that `swig` is missing. Do we need to add `swig`, `swig2.0`, or `swig3.0` to the `packages` section?

File: .travis.yml [hunk 1]
```diff
@@ -14,15 +14,18 @@ os:
...
+    env: EXTRA_BUILD_FLAGS="-DPYTHON_EXECUTABLE=/usr/bin/python3 -DUSE_SYSTEM_HDF5=ON -DUSE_SYSTEM_FFTW3=ON" CC=gcc-5 CXX=g++-5 PYVER=3
+  - python: 2.7
+    env: EXTRA_BUILD_FLAGS="-DUSE_SYSTEM_HDF5=ON -DUSE_SYSTEM_FFTW3=ON" CC=gcc-5 CXX=g++-5 PYVER=
 env:
   global:
-    - BUILD_FLAGS="-DUSE_SYSTEM_Boost=ON -DCMAKE_BUILD_TYPE=Release"
-  matrix:
-    - EXTRA_BUILD_FLAGS="-DPYTHON_EXECUTABLE=/usr/bin/python3 -DUSE_SYSTEM_HDF5=ON -DUSE_SSYTEM_SWIG=ON -DUSE_SYSTEM_FFTW3=ON" CC=gcc-5 CXX=g++-5 PYVER=3
-    - EXTRA_BUILD_FLAGS=" -DUSE_SSYTEM_SWIG=ON -DUSE_SYSTEM_HDF5=ON -DUSE_SYSTEM_FFTW3=ON" CC=gcc-5 CXX=g++-5 PYVER=
+    - BUILD_FLAGS="-DUSE_SYSTEM_SWIG=ON -DUSE_SYSTEM_Boost=ON -DCMAKE_BUILD_TYPE=Release"
```
Comment (KrisThielemans) [inline .travis.yml]: I guess with this you force USE_SYSTEM_SWIG=On for all builds. in the previous set-up, one build used the system swig, while the other used the superbuild to make swig. I'd prefer that. Also, we'd need it for the osx build (unless we'd "brew install swig" of course).

Comment (casperdcl) [reply .travis.yml]: I haven't changed what it looked like the old set-up was trying to do. USE_SSYTEM_SWIG=ON appeared in all `EXTRA_BUILD_FLAGS` (including the commented out osx version) so I corrected the typo and moved it to global `BUILD_FLAGS`

Comment (casperdcl) [reply .travis.yml]: I've added `swig` to `addons: apt: packages`, seems to be building fine for linux

Comment (KrisThielemans) [reply .travis.yml]: hmm. yes indeed. I guess we were trying the simplest options first. I'll try some changes.

File: .travis.yml [hunk 2]
```diff
@@ -17,15 +17,15 @@ os:
 matrix:
   include:
   - python: 3.5
-    env: EXTRA_BUILD_FLAGS="-DPYTHON_EXECUTABLE=/usr/bin/python3 -DUSE_SYSTEM_HDF5=ON -DUSE_SYSTEM_FFTW3=ON" CC=gcc-5 CXX=g++-5 PYVER=3
+    env: EXTRA_BUILD_FLAGS="-DUSE_SYSTEM_HDF5=ON -DUSE_SYSTEM_FFTW3=ON" CC=gcc-5 CXX=g++-5
```
Comment (casperdcl) [inline .travis.yml]: I dodn't look at all the cmake code, so wasn't sure if it was ok to remove `PYTHON_EXECUTABLE` and `PYVER`.

**Pull request merged**